doc/api.rst

```diff
@@ -538,6 +539,21 @@ Runner class
 
       See the :ref:`bench_func() example <bench_func_example>`.
 
+   .. method:: bench_async_func(name, func, \*args, inner_loops=None, metadata=None)
+
+      Benchmark the function ``await func(*args)`` in asyncio event loop.
```

Comment:
Compared to ``asyncio.run()``, the risk is that func() coroutine leaves pending events which will be handled by the following benchmark, or between iterations of the same benchmark value. I'm thinking at sockets and loop.call_later().

Should the documentation warns about these issues? The developer should pay attention to no "leak" any pending event in func() coroutine.